mp/src/game/client/hud_chat.cpp

```diff
@@ -54,6 +54,7 @@ void CHudChat::StartMessageMode()
     SetKeyBoardInputEnabled(true);
 
     m_pChatContainer->StartMessageMode(MESSAGE_MODE_HUD);
+    m_pChatContainer->SetMouseInputEnabled(true);
```

Comment:
Something to test, would the container always having mouse input be better here? The HUD element having it takes mouse input away from the game, but would otherwise be the only thing needing toggled here.

Reply:
Something like what I just pushed? Seems like it can always have mouse input from constructor and works just fine.

Reply:
Yes, so the other line can be removed too then?

Reply:
It indeed can. Needed it before but not anymore with this change.